### PR TITLE
add privacy policy

### PR DESCRIPTION
**Pull request by MattHJensen**
@hdoupe, could you look this over? 

I will open an issue describing where this should be accessible from.

### CODE REVIEW

File: docs/PRIVACY.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+### Analytics Services Provided By Others
+
+We may allow others to provide analytics services on our behalf. These entities may use cookies, web beacons and other technologies to collect information about your use of our Services and other websites, including your IP address, web browser, pages viewed, time spent on pages, links clicked and conversion information. This information may be used by COMP and others to, among other things, analyze and track data, determine the popularity of certain content, deliver content targeted to your interests and better understand your online activity.
+
+## Your Choices
+
+### Account Information
+
+You may update, correct or delete information about you at any time by logging into your online account. If you wish to delete your account, please email us at admin@compmodels.com, but note that we may retain certain information as required by law or for legitimate business purposes. We may also retain cached or archived copies of information about you for a certain period of time.
```
Comment (hdoupe) [inline docs/PRIVACY.md]: There's a "delete account" option in the user's profile.

File: docs/SUBPROCESSORS.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# COMP Data Sub-processors
+
+| **Entity**          | **Purpose**               | **Entity Country**| **Website**            |
+|---------------------|---------------------------|-------------------|------------------------|
+| Amazon Web Services | Cloud services provider   | United States     | https://aws.amazon.com |
+| SendGrid            | Email services provider   | United States     | https://sendgrid.com/  |
+| Stripe              | Billing services provider | United States     | https://stripe.com     |
```
[the remark this answers is not in the document]
Comment (hdoupe) [reply docs/SUBPROCESSORS.md]: [PaperTrail](https://papertrailapp.com/) is used via a Heroku plugin for storing log data, too.

File: docs/PRIVACY.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+### Changes to this Privacy Policy
+
+We may change this Privacy Policy from time to time. If we make changes, we will notify you by revising the date at the bottom of the policy and, in some cases, we may provide you with additional notice (such as adding a statement to our homepage or sending you a notification). We encourage you to review the Privacy Policy whenever you access our Services to stay informed about our information practices and the ways you can help protect your privacy.
+
+Contact Information
+To ask questions or comments about this Privacy Policy or our privacy practices please email us at admin@compmodels.com or write us at Compute Tooling, Inc. 213 E Highland Ave., Atlantic Highlands, NJ 07716. 
+
+This version of the Privacy Policy was last updated on December 22, 2018 and historic versions are archived here: https://github.com/comp-org/comp/commits/master/docs/PRIVACY.md.
```
Comment (hdoupe) [inline docs/PRIVACY.md]: I think the link should be: https://github.com/comp-org/comp/blob/master/docs/PRIVACY.md

**Comment by hdoupe** [comment]
@MattHJensen thanks for knocking this out. This looks good to me once the comments have been resolved.

File: docs/PRIVACY.md (hunk 3 above)
Comment (MattHJensen) [reply docs/PRIVACY.md]: Your link takes me to the latest, but I want to link to the version archives, which is the link I have. See, https://github.com/comp-org/comp/commits/master/docs/TERMS.md and https://github.com/comp-org/comp/blob/master/docs/TERMS.md

Comment (hdoupe) [reply docs/PRIVACY.md]: Ah, I see. That makes sense.

**Comment by MattHJensen** [comment]
Latest commit ought to resolve @hdoupe concerns. Merging.

**Pull request merged**